test/es6/ES6Promise.js

```diff
@@ -221,6 +229,16 @@ var tests = [
             assert.throws(function() { Promise.allSettled.call(Math.sin); }, TypeError, "Promise.allSettled throws when this parameter is a non-constructor", "Function expected");
         }
     },
+    {
+        name: "Promise.any throwing behavior",
+        body: function () {
+            assert.throws(function() { Promise.any.call(); }, TypeError, "Promise.any throws when called with no this parameter", "Function expected");

```

Comment:
Should add coverage here for a string, a float and an int. (Currently these will hard crash - I've noted the fix above)